db/migrate/20231027144605_add_unit_amount_to_fees.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class AddUnitAmountToFees < ActiveRecord::Migration[7.0]
+  def up
+    change_column :fees, :unit_amount_cents, :bigint, null: false, default: 0
```

Comment:
Rails/BulkChangeTable: You can use `change_table :fees, bulk: true` to combine alter queries. (https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-change_table, https://api.rubyonrails.org/classes/ActiveRecord/ConnectionAdapters/Table.html)